Makefile

```diff
@@ -34,12 +34,28 @@ $(info $(shell printf "%-21s = %-10s\n" "KUBE_CONTROLLERS_VER" $(KUBE_CONTROLLER
 $(info $(shell printf "%-21s = %-10s\n" "TYPHA_VER" $(TYPHA_VER)))
 ##############################################################################
 
-serve:
-	docker run --rm -ti -e JEKYLL_UID=`id -u` -p 4000:4000 -v $$PWD:/srv/jekyll jekyll/jekyll:$(JEKYLL_VERSION) jekyll serve --incremental $(CONFIG)
+serve: bin/helm
+	# We have to override JEKYLL_DOCKER_TAG which is usually set to 'pages'. 
+	# When set to 'pages', jekyll starts in safe mode which means it will not
+	# load any plugins. Since we're no longer running in github-pages, but would
+	# like to use a docker image that comes preloaded with all the github-pages plugins,
+	# its ok to override this variable.
+	docker run --rm \
+	  -v $$PWD/bin/helm:/usr/local/bin/helm:ro \
+	  -v $$PWD:/srv/jekyll \
+	  -e JEKYLL_DOCKER_TAG="" \
```

Comment:
Is there documentation on this JEKYLL_DOCKER_TAG somewhere? What does this do?

Reply:
yes, 5 lines above this.

Reply:
I see the comment... but is this a variable we've defined? Something the image supports? A google search didn't show me much. 

I found this: https://github.com/envygeeks/jekyll-docker/blob/master/repos/jekyll/copy/all/usr/jekyll/bin/default-args#L18

But, that simply tells me the same thing your comment above does. Wonder if it triggers any other behavior that we may or may not want, is all.

Reply:
It's defined and used within the jekyll docker image scripts: https://github.com/envygeeks/jekyll-docker/blob/70daad404f03e287ed04cf2192a3befeef102913/repos/jekyll/copy/all/usr/jekyll/bin/default-args#L18

From my analysis, it's only used to trigger safe mode.